client/src/views/users/InvitationsListPopover.vue

```diff
@@ -0,0 +1,172 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <div class="invitations-list-container">
+    <div class="invitations-list">
+      <div class="invitations-list-header">
+        <ion-text class="invitations-list-header__title title-h4">
+          {{ $t('invitationList.title') }}
+        </ion-text>
+        <span class="invitations-list-header__counter body-sm">
+          {{ invitations.length }}
+        </span>
+        <ion-button
```

Comment:
Is an "invite user" button necessary here ? This popover should notify the user of pending invitations only imo, adding this functionality seems weird, moreso since it's inaccessible if there are no current invitations.